staging/src/k8s.io/kubelet/pkg/apis/stats/v1alpha1/types.go

```diff
@@ -83,6 +83,11 @@ type RuntimeStats struct {
 	// Usage here refers to the total number of bytes occupied by images on the filesystem.
 	// +optional
 	ImageFs *FsStats `json:"imageFs,omitempty"`
+	// Stats about the underlying filesystem where container's writeable layer is stored.
```

Comment:
I don't think you need to add in v1alpha1 (we're probably due soon to drop support for it @SergeyKanzhelev )

Reply:
IMO that should be a separate change than this PR.  

There is only v1alpha1 for this object so if we want to drop support for this, I think we would want to move this all in one go.

Reply:
yeah definitely a separate change

Reply:
When we promote cri stats to beta, I think this should be part of that KEP.  Are we tracking that there?

Reply:
I thought this was CRI v1alpha1, not the stats, so actually we do need this and probably won't drop support